### PR TITLE
feat: add checkVersion option for mobile:installApp

### PR DESCRIPTION
**Pull request by KazuCocoa**
https://github.com/appium/appium-android-driver/pull/915 for uia2.
I noticed missing args about `mobile: removeApp` in the readme, so this pr includes it as well.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -812,6 +814,7 @@ allowTestPackages | boolean | no | Set to true in order to allow test packages i
 useSdcard | boolean | no | Set to true to install the app on sdcard instead of the device memory. false by default | true
 grantPermissions | boolean | no | Set to true in order to grant all the permissions requested in the application's manifest automatically after the installation is completed under Android 6+. The targetSdkVersion in the application manifest must be greater or equal to 23 and the Android version on the device under test must be greater or equal to Android 6 (API level 23) to grant permissions. Applications whose targetSdkVersion is lower than or equal to 22 must be reisntalled to grant permissions for Android 6+ devices. false by default | true
 replace | boolean | no | Set it to false if you don't want the application to be upgraded/reinstalled if it is already present on the device, but throw an error instead. true by default | false
+checkVersion | boolean | no | Set to true, in order to skip the application installation if the device under test has greater or equal to the application version. | true
```
Comment (mykola-mokhnach) [inline README.md:817]: Please also mention that by default an exception is thrown upon install if the same or an older app version is already installed

Comment (KazuCocoa) [reply README.md:817]: I haven't tested with an older one, but the same version was installed successfully with `checkVersion` `false` (or not given) as default as `-r`. The behavior may depend on the `replace`.

Comment (mykola-mokhnach) [reply README.md:817]: ok, then only mention the older one

Comment (KazuCocoa) [reply README.md:817]: Ensured that INSTALL_FAILED_VERSION_DOWNGRADE error occurs only for downgrade even with `-r`

Comment (KazuCocoa) [reply README.md]: btw, the same version succeeded in installing without `-r` as well

**Review by mykola-mokhnach — APPROVED** [review]

**Pull request merged**